src/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Footer from './components/Common/Footer';
+import Navbar from './components/Common/Navbar';
+import Path from './Paths/Path';
+
+function App() {
+  return (
+    <>
+      <Navbar/>
+      <Path/>
```

Comment:
path-> route.. path file should at root of src

Reply:
Renamed Path to Route and added in Src root